api/pyproject.toml

```diff
@@ -66,3 +66,109 @@ MOCK_SWITCH = "true"
 CODE_MAX_STRING_LENGTH = "80000"
 CODE_EXECUTION_ENDPOINT="http://127.0.0.1:8194"
 CODE_EXECUTION_API_KEY="dify-sandbox"
+
+
+[tool.poetry]
+name = "dify-api"
+version = "0.6.8"
+description = ""
+authors = ["Dify <hello@dify.ai>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+beautifulsoup4 = "^4.12.2"
+flask = "^3.0.1"
+flask-sqlalchemy = "^3.0.5"
+sqlalchemy = "^2.0.29"
+flask-compress = "^1.14"
+flask-login = "^0.6.3"
+flask-migrate = "^4.0.5"
+flask-restful = "^0.3.10"
+flask-cors = "^4.0.0"
+gunicorn = "^22.0.0"
+gevent = "^23.9.1"
+openai = "^1.29.0"
+tiktoken = "^0.7.0"
+psycopg2-binary = "^2.9.6"
+pycryptodome = "3.19.1"
+python-dotenv = "1.0.0"
+authlib = "1.2.0"
+boto3 = "^1.28.17"
+tenacity = "^8.2.2"
+cachetools = "^5.3.0"
+weaviate-client = "^3.21.0"
+mailchimp-transactional = "^1.0.50"
+scikit-learn = "1.2.2"
+sentry-sdk = {version = "^1.39.2", extras = ["flask"]}
+sympy = "1.12"
+jieba = "0.42.1"
+celery = "^5.3.6"
+redis = {version = "^5.0.3", extras = ["hiredis"]}
+openpyxl = "3.1.2"
+chardet = "^5.1.0"
+python-docx = "^1.1.0"
+pypdfium2 = "^4.17.0"
+resend = "^0.7.0"
+pyjwt = "^2.8.0"
+anthropic = "^0.23.1"
+newspaper3k = "0.2.8"
+wikipedia = "1.4.0"
+readabilipy = "0.2.0"
+google-ai-generativelanguage = "0.6.1"
+google-api-core = "^2.18.0"
+google-api-python-client = "^2.90.0"
+google-auth = "^2.29.0"
+google-auth-httplib2 = "0.2.0"
+google-generativeai = "0.5.0"
+google-search-results = "2.4.2"
+googleapis-common-protos = "1.63.0"
+google-cloud-storage = "2.16.0"
+google-cloud-aiplatform = "^1.52.0"
+replicate = "^0.22.0"
+websocket-client = "^1.7.0"
+dashscope = {version = "^1.17.0", extras = ["tokenizer"]}
+huggingface-hub = "^0.16.4"
+transformers = "^4.35.0"
+tokenizers = "^0.15.0"
+pandas = "1.5.3"
+xinference-client = "0.9.4"
+safetensors = "^0.4.3"
+zhipuai = "1.0.7"
+werkzeug = "^3.0.1"
+pymilvus = "2.3.1"
+qdrant-client = "1.7.3"
+cohere = "^5.2.4"
+pyyaml = "^6.0.1"
+numpy = "^1.26.4"
+unstructured = {version = "^0.10.27", extras = ["docx", "epub", "md", "msg", "ppt", "pptx"]}
+bs4 = "^0.0.1"
+markdown = "^3.5.1"
+httpx = {version = "^0.24.1", extras = ["socks"]}
+matplotlib = "^3.8.2"
+yfinance = "^0.2.35"
+pydub = "^0.25.1"
+gmpy2 = "^2.1.5"
+numexpr = "^2.9.0"
+duckduckgo-search = "5.2.2"
+arxiv = "2.1.0"
+yarl = "^1.9.4"
+twilio = "^9.0.4"
+qrcode = "^7.4.2"
+azure-storage-blob = "12.9.0"
+azure-identity = "1.15.0"
+lxml = "5.1.0"
+xlrd = "^2.0.1"
+pydantic = "^1.10.0"
+pgvecto-rs = "0.1.4"
+firecrawl-py = "0.0.5"
+oss2 = "2.15.0"
+pgvector = "0.2.5"
```

Comment:
Please update the list by appending recently added dependencies in `requirements.txt` like
```
pymysql
tidb-vector
google-cloud-aiplatform
vanna[postgres,mysql,clickhouse,duckdb]

Reply:
Fixed